pkg/maps/lbmap/lbmap.go

```diff
@@ -398,6 +399,14 @@ func (*LBBPFMap) UpdateSourceRanges(revNATID uint16, prevSourceRanges []*cidr.CI
 
 	srcRangeMap := map[string]*cidr.CIDR{}
 	for _, cidr := range sourceRanges {
+		// k8s api server does not catch the IP family mismatch, so we need to catch it here
+		if ip.IsIPv6(cidr.IP) == !ipv6 {
+			log.WithFields(logrus.Fields{
+				logfields.ServiceID: revNATID,
+				logfields.CIDR:      cidr,
+			}).Warn("Source range's IP family does not match with the LB's")
```

Comment:
One more nit - could you add to the log message ". Ignoring the source range CIDR", so that users won't think that the whole src range configuration is ignored.

Reply:
Updated.